harvard-lts/CURIOSity
# Upgrade ruby

**Moves us to Ruby3**
* * *

# What does this Pull Request do?
Moves to Ruby3 and also using phusion image instead of alpine, also some library bumps for security patches.

# How should this be tested?

Spin up this branch and make sure everything still works!

This is also deployed to dev, as the home dir name changed so this will have an associated IF PR.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
